exercises/message/src/starks/prover.rs

```diff
@@ -744,6 +744,12 @@ where
         .collect()
 }
 
+strct StoneProverTranscript {
+    counter: usize,
+    spare_bytes: Vec<u8>,
+    hash: dyn Transcript,
+}
+
```

Comment:
This is odd, strct is not a struct. But somehow it compiled ? Is this used ?

Reply:
Also if counter is not indexing anything I'd just do it a u32

Reply:
that code is there by mistake (exercises). Already removed it. And also make the counter a `u32`.